src/unspent_store.py

```diff
@@ -0,0 +1,204 @@
+import asyncio
+from typing import Dict, Optional, List
+import aiosqlite
+from src.types.full_block import FullBlock
+from src.types.hashable.Coin import Coin, CoinName
+from src.types.hashable.Unspent import Unspent
+from src.types.header_block import HeaderBlock
+from src.types.sized_bytes import bytes32
+from src.util.ints import uint32
+
+
+class DiffStore:
+    header: HeaderBlock
+    diffs: Dict[CoinName, Unspent]
+
+    @staticmethod
+    async def create(head: HeaderBlock, diffs: Dict[CoinName, Unspent]):
+        self = DiffStore()
+        self.header = head
+        self.diffs = diffs
+        return self
+
+
+class UnspentStore:
+    """
+    This object handles unspent coins in DB.
+    Coins from genesis to LCA are stored on disk db, coins from lca to head are stored in DiffStore object for each tip.
+    When blockchain notifies UnspentStore of new LCA, LCA is added to the disk db,
+    DiffStores are updated/recreated. (managed by blockchain.py)
+    """
+    db_name: str
+    unspent_db: aiosqlite.Connection
+    # Whether or not we are syncing
+    sync_mode: bool = False
```

Comment:
I imagined it will be used when adding to unspent store is optimized during sync(Not creating diffs between LCA and heads), but I guess it's not really necessary as logic for that will be in blockchain.py